xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/XWiki.java

```diff
@@ -109,7 +109,6 @@
 import org.xwiki.container.servlet.HttpServletUtils;
 import org.xwiki.context.Execution;
 import org.xwiki.edit.EditConfiguration;
-import org.xwiki.extension.event.ExtensionInitializedEvent;
```

Comment:
You might want to reset any change made to this file since they are not related to your pull request.

Reply:
I didn't notice this one and yes for sure.